### DISCUSSION TITLE
Can we connect PAC's server to MONAILabel for annotating the CT Images?

### DISCUSSION THREAD

**Discussion by hiq-ayeddula** [Project-MONAI/MONAILabel#1647] [Q&A]
We have a PAC's server and is there any possibility of connecting our PAC's server to MONAILable for annotating the CT images?
If yes, please provide any documentation/videos for the same.

**Comment by diazandr3s** [comment]
Hi @hiq-ayeddula, 

This is a great question.
For this, I'd recommend checking the XNAT+MONAI Label integration: https://wiki.xnat.org/xnat-ohif-viewer/release-notes#ReleaseNotes-v3.4.0
Using XNAT you could fetch DICOM studies from the PACS and then organize them to train or run inference using MONAI Label.
You may already know this, but [XNAT](https://wiki.xnat.org/documentation/) could be a solution to your workflow. It offers dataset and user management - perfect to work with MONAI Label.

Hope this helps,


P.D. Happy to further discuss your use case over a video call.

**Reply by RaviTeja-highiq** [reply]
@diazandr3s I am a colleague of Anil and would like to take you up on the offer of connecting over a video call. 
Thankyou...

**Reply by diazandr3s** [reply]
Superb! Let's chat @RaviTeja-highiq. Do you have an email address I can contact? Otherwise, let's chat on LinkedIn.

**Reply by RaviTeja-highiq** [reply]
ravi@myisha.ai
